sandbox_externalkey_unsupported.go

```diff
@@ -0,0 +1,42 @@
+// +build !linux
+
+package libnetwork
+
+import (
+	"io"
+	"net"
+)
+
+// processSetKeyReexec is a private function that must be called only on an reexec path
+// It expects 3 args { [0] = "libnetwork-setkey", [1] = <container-id>, [2] = <controller-id> }
+// It also expects libcontainer.State as a json string in <stdin>
+// Refer to https://github.com/opencontainers/runc/pull/160/ for more information
+func processSetKeyReexec() {
+}
+
+// SetExternalKey provides a convenient way to set an External key to a sandbox
+func SetExternalKey(controllerID string, containerID string, key string) error {
+	return nil
```

Comment:
I've made what's used outside those files to return `nil` as a no-op and return a `ErrNotImplemented` otherwise, I hope it makes sense now